src/libraries/System.Text.Json/src/System/Text/Json/Nodes/JsonValue.cs

```diff
@@ -20,11 +21,11 @@ public abstract partial class JsonValue : JsonNode
         /// <returns>
         ///   The new instance of the <see cref="JsonValue"/> class that contains the specified value.
         /// </returns>
-        /// <typeparam name="T">The type of value to be added.</typeparam>
-        /// <param name="value">The value to add.</param>
+        /// <typeparam name="T">The type of value to create.</typeparam>
+        /// <param name="value">The value to create.</param>
         /// <param name="options">Options to control the behavior.</param>
         /// <returns>The new instance of the <see cref="JsonValue"/> class that contains the specified value.</returns>
-        public static JsonValue? Create<T>(T? value, JsonNodeOptions? options = null)
+        public static JsonValue? Create<[DynamicallyAccessedMembers(JsonHelpers.MembersAccessedOnRead)]T>(T? value, JsonNodeOptions? options = null)
```

Comment:
I thought we were going to mark this as `RequiresUnreferencedCode`.

Reply:
Only or in addition to DynamicallyAccessedMembers? I assume the same applies to the `Add()`

Reply:
For now, it needs to be in addition to `DynamicallyAccessedMembers`. But the `DynamicallyAccessedMembers` will be removed with https://github.com/dotnet/runtime/issues/52268.